### PR TITLE
add functional test site maps generator

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-front-demo/pull/272
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/633

### CODE REVIEW

**Comment by itkg-nanne** [comment]
:+1:

**Comment by alavieille** [comment]
@ngilain updated test

File: src/OpenOrchestra/Tests/Functional/FrontBundle/Command/OrchestraGenerateSiteMapsCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    /**
+     * Provide sites aliases
+     */
+    public function provideSiteAlias()
+    {
+        return array(
+            array('3'),
+        );
+    }
+}
```
Comment (ngilain) [inline src/OpenOrchestra/Tests/Functional/FrontBundle/Command/OrchestraGenerateSiteMapsCommandTest.php]: @alavieille : can you also test the definition and the command line such as in https://github.com/open-orchestra/open-orchestra-front-bundle/blob/master/FrontBundle/Tests/Command/OrchestraGenerateSitemapCommandTest.php ?

**Comment by ngilain** [comment]
:+1:

**Pull request merged**